website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -133,10 +135,10 @@ Create a `service-defaults` custom resource for the external service:
 Apply the `ServiceDefaults` resource with `kubectl apply`:
 
 ```shell-session
-$ kubectl apply --filename service-defaults.yaml
+$ kubectl apply --filename serviceDefaults.yaml
```

Comment:
not a problem but the filename is fine to be `-` separated and is pretty normal for k8s yaml files